nav2_bringup/launch/bringup_launch.py

```diff
@@ -100,14 +127,21 @@ def generate_launch_description():
                               'params_file': params_file,
                               'use_lifecycle_mgr': 'false'}.items()),
 
+        Node(
+            condition=IfCondition(use_composition),
+            package='nav2_bringup',
+            executable='composed_bringup',
+            output='screen',
+            parameters=[configured_params, {'autostart': autostart}],
+            remappings=remappings),
+
         IncludeLaunchDescription(
             PythonLaunchDescriptionSource(os.path.join(launch_dir, 'navigation_launch.py')),
+            condition=IfCondition(PythonExpression(['not ', use_composition])),
             launch_arguments={'namespace': namespace,
                               'use_sim_time': use_sim_time,
                               'autostart': autostart,
-                              'params_file': params_file,
-                              'use_lifecycle_mgr': 'false',
```

Comment:
Why remove these 2 other launch arguments?

Reply:
i only remove `use_lifecycle_mgr`, because it's unused in `localization_launch.py`. do you think it's a reasonable change? if not, i will change back.

Reply:
This is for navigation_launch.py though?

Reply:
`use_lifecycle_mgr` is also unused in `navigation_launch.py` , i think i need remove it, right?

Reply:
Good point, I tracked it down, its from an old feature we deprecated.  Nice catch